main.go

```diff
@@ -8,18 +8,16 @@ import (
 	"strings"
 	"time"
 
-	"github.com/sirupsen/logrus"
-
-	_ "net/http/pprof"
```

Comment:
Exposes debug information as side-effect on the server port. This information already exposed on the profile address in telemetry.